news/feature-5063.md

```diff
@@ -0,0 +1,13 @@
+`syslog-parser`: Added the `no-piggyback-errors` and the `piggyback-errors` flags to control whether the message retains the original message or not. By default the old behaviour/`no-piggyback-errors` flag is active, modifing the original message on error.
+
+- `no-piggyback-errors`: On failure the original message, will be left as it was before parsing, the value of `$MSGFORMAT` will be set to `syslog-error`, and a tag will be placed on the message roccesponding to the parser's failure.
+- `piggyback-errors`: On failure the old behaviour is used(clearing the netire message, then syslog-ng will generate a new message in place of the old one describing the parser's error).
```

Comment:
Just some small issues
- missing space before the `(`
- small typo `netire`
- unneeded comma before then

Reply:
i fixed them